src/components/ProjectList.js

```diff
@@ -11,7 +11,12 @@ export default function ProjectList({ projectList }) {
         textColor={'var(--copper-ultralight)'}
       />
       <StyledProjectList>
-        <ProjectListItem projectList={projectList} />
+        {projectList.map((projectData, index) => (
```

Comment:
Sehr gut, dass du es auf diese Ebene geholt hast. Ich erinnere mich, dass wir das im Online Shop damals auch im "Item" gemacht hatten und als Feedback bekommen haben, dass es eine Ebene höher muss 👍

Reply:
Ja daran hatte ich mich auch wieder erinnert nachdem ich es in ein paar Code Reviews anders gesehen hatte! 😄